libs/core-ui/src/lib/util/getDependencyChartOptions.ts

```diff
@@ -49,6 +49,7 @@ export function getDependencyChartOptions(
         color: pointColor,
         data,
         name: "",
+        turboThreshold: 0,
```

Comment:
This disables a bunch of checks to make it faster for larger amounts of data, right? Is that safe? I couldn't find anything on that in the documentation.

Reply:
Giving that we only take 5k points, it is fine for now.
Gaurav's big data work will rewrite this piece of code.